src/Aspire.Hosting.Testing/DistributedApplicationFactory.cs

```diff
@@ -1,20 +1,20 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
-using System.Diagnostics;
+
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using System.Diagnostics;
 
 namespace Aspire.Hosting.Testing;
 
 /// <summary>
 /// Factory for creating a distributed application for testing.
 /// </summary>
-/// <typeparam name="TEntryPoint">
-/// A type in the entry point assembly of the target Aspire AppHost. Typically, the Program class can be used.
-/// </typeparam>
-public class DistributedApplicationFactory<TEntryPoint> : IDisposable, IAsyncDisposable where TEntryPoint : class
```

Comment:
Are we going to continue to validate `where TEntryPoint : class`. Was that important?

Reply:
Seems just a carry over from `WebApplicationFactory<TEntryPoint>` which this was based on but I can't see a good reason why it's restricted there too. I don't think it matters too much. We can always loosen the generic type parameter constraint if we ever get feedback on it and folks aren't willing to use the `Type` overload.